core/src/main/java/org/apache/carbondata/core/statusmanager/LoadMetadataDetails.java

```diff
@@ -241,26 +248,28 @@ private long convertTimeStampToLong(String factTimeStamp) {
    * @return
    */
   public Long getTimeStamp(String loadStartTime) {
-    Date dateToStr = null;
     try {
-      dateToStr = parser.parse(loadStartTime);
-      return dateToStr.getTime() * 1000;
-    } catch (ParseException e) {
-      LOGGER.error("Cannot convert" + loadStartTime + " to Time/Long type value" + e.getMessage());
-      return null;
+      // for new loads the factTimeStamp will be long string
```

Comment:
In future for maintenance it is not easy to understand what new load is, can you add more background information for better maintenance.

Reply:
Fixed Added the detailed comment at class level

Reply:
Do not mention "new loads" in line 275. Add comment in line 279, mentioning it is the processing for existing table before carbon 1.3

Reply:
fixed